### PR TITLE
[tx_hash label] [Easy] Added new columns and fixed query in tx_hash label

### PR DESCRIPTION
**Pull request by gentrexha**
I've added some of the new label columns to the tx_hash spell. In retrospect, it would've been a good idea to discuss this label as part of the [Relaunching Labels discussion](https://github.com/duneanalytics/spellbook/discussions/2634) and discussing the possibility of this being a separate category of labels.

### CODE REVIEW

**Comment by gentrexha** [comment]
Tagging @bh2smith for internal review

File: models/cow_protocol/tx_hash_labels/stable_to_stable/ethereum/cow_protocol_tx_hash_labels_stable_to_stable_ethereum.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@
 with
  stable_to_stable_trades as (
     select
-        distinct tx_hash
+        tx_hash
```
Comment (bh2smith) [inline models/cow_protocol/tx_hash_labels/stable_to_stable/ethereum/cow_protocol_tx_hash_labels_stable_to_stable_ethereum.sql:10]: What is the reason for this change? Was it wrong before?

**Review by bh2smith — COMMENTED** [review]
Not really sure why we are removing distinct and array here though...

**Comment by gentrexha** [comment]
@Hosuke Can I get this merged because I need it for the other PRs?

**Comment by Hosuke** [comment]
> @Hosuke Can I get this merged because I need it for the other PRs?

Noted.

**Review by Hosuke — APPROVED** [review]
LGTM.✅

Thanks @gentrexha , also please review the discussion here to see if we need to make any change to this PR: https://github.com/duneanalytics/spellbook/discussions/2810#discussioncomment-5092164

**Comment by jeff-dude** [comment]
i'd prefer to see the linked discussion above to be resolved prior to merging all of these tx_hash label PRs

**Comment by gentrexha** [comment]
> per linked GH discussion, we will merge the open PRs, then @gentrexha will fit this into labels new structure. any new questions along the way can be raised there.

Can I add two more types of tx_hash labels and merge everything? We want to get this over this line before the end of this week, and I would appreciate it if I could start analyzing the results before moving everything.

**Comment by jeff-dude** [comment]
> > per linked GH discussion, we will merge the open PRs, then @gentrexha will fit this into labels new structure. any new questions along the way can be raised there.
> 
> Can I add two more types of tx_hash labels and then merge everything? We want to get this over this line before the end of this week, and I would appreciate it if I could start analyzing the results before moving everything.

yes, happy to do so as soon as you fix the issues on other PR i commented on recently with failure on unique columns. i'd imagine that happens on other similar PRs too.
